### ISSUE TITLE
Detect the usage of high contrast mode

### ISSUE THREAD

**Issue by lunars97** [digitalfabrik/integreat-app#3190]
### Describe the Problem
Ideally it would be nice to have it on android as well, but android does not have this function yet (see this closed PR #2795 discussion). iOS might have this function to detect the enabled high contrast mode in system. 
### Describe Your Preferred Solution
As a user it would be nice to continue to use the system high contrast mode on the app without manually switching the toggle on side menu.
### Describe Possible Alternatives
Leave as it is because it is not that urgent.

**Comment by steffenkleinle** [comment]
This is on-hold for now as long as the high contrast mode is still a beta feature (and we have problems with some icons, see #3271).
